resources/js/Components/Table/CharacterTable/CharacterTable.tsx

```diff
@@ -167,12 +167,12 @@ const CharacterTable = ({
                                 page * rowsPerPage,
                                 page * rowsPerPage + rowsPerPage,
                             )
-                            .map((row: RowData, index: number) => {
-                                const isItemSelected = isSelected(row.cname)
+                            .map((row: CharacterRowData, index: number) => {
+                                const isItemSelected = isSelected(row.name)
```

Comment:
can you also refactor the isSelected function to make it use `.includes` instead